### ISSUE TITLE
Docset Request: Zeal?

### ISSUE THREAD

**Issue by Rohaq** [zealdocs/zeal#1112]
It's a bit ironic that Zeal's own local documentation is somewhat lacking - There's not even a link to any help documentation from inside the "Help" menu! This meant that in order to find the syntax to search individual Docsets, I had to look up the details on the `README.md` on Github - which required going online.

Zeal's documentation might be a little... short, but it would make sense to make it available without needing an online connection - maybe even give it its own Docset accessible within Zeal?

**Comment by trollixx** [comment]
Yes, a built-in documentation would definitely be useful for everyone.

**Comment by malaire** [comment]
Also keyboard shortcuts seems to be documented only online: https://zealdocs.org/usage.html

**Comment by xerus2000** [comment]
> Also keyboard shortcuts seems to be documented only online: https://zealdocs.org/usage.html

I guess this could actually be part of the "Welcome page" when opening Zeal
